src/catalog/src/durable/impls/migrate.rs

```diff
@@ -202,11 +221,12 @@ impl OpenableDurableCatalogState for CatalogMigrator {
     }
 
     async fn is_initialized(&mut self) -> Result<bool, CatalogError> {
-        let tombstone = self.get_tombstone().await?;
-        if tombstone == Some(true) {
-            self.openable_persist.is_initialized().await
-        } else {
+        match self.target {
```

Comment:
extraneous?

Reply:
Oops, yeah I didn't mean to touch this function at all.